src/gui/qgisinterface.h

```diff
@@ -970,6 +970,14 @@ class GUI_EXPORT QgisInterface : public QObject
      */
     virtual void preloadForm( const QString &uifile ) = 0;
 
+    /**
+     * This will perforom a search in the locator bar
```

Comment:
"This will perform a search in the locator bar by setting the line edit text to \a searchText and automatically displaying any  results."